tests/unittests/CoreGraphics.Drawing/CGPathDrawingTests.cpp

```diff
@@ -663,4 +663,61 @@ DRAW_TEST_F(CGPath, AddLinesWithoutMove, UIKitMimicTest<>) {
     CGContextStrokePath(context);
 
     CGPathRelease(thepath);
+}
+
+DRAW_TEST_F(CGPath, WidenTest_Stroking, UIKitMimicTest<>) {
+    CGContextRef context = GetDrawingContext();
+    applyBounds = GetDrawingBounds();
```

Comment:
applyBounds was a static Rect used earlier in the file. It should not be used here. Fixing + other tests.